explorer/src/pages/AccountDetailsPage.tsx

```diff
@@ -517,17 +517,17 @@ function getAnchorTabs(pubkey: PublicKey, account: Account) {
     ),
   });
 
-  const anchorAccountTab: Tab = {
+  const accountDataTab: Tab = {
     slug: "anchor-account",
-    title: "Anchor Account",
+    title: "Anchor Data",
     path: "/anchor-account",
```

Comment:
Is it worth changing this slug, since this feature is so new and it's unlikely that folks have any of this bookmarked yet?

Reply:
I actually thought I reverted to the original :sweat_smile:  (I reverted sort of manually looking at the diff from the first commit). Feel free to revert as needed.